script.js

```diff
@@ -0,0 +1,55 @@
+// Brand Currency form -------------
+const brandObject = {
+  timestamp: Date.now(),
+  base: "EUR",
+  date: new Date().toLocaleDateString(),
+  rates: {},
+};
+
+document
+  .getElementById("brand-currency-form")
+  .addEventListener("submit", brandCurrency);
+
+function brandCurrency(event) {
+  event.preventDefault();
+
+  const baseCurrency = document.getElementById("base-currency").value;
+  brandObject.base = baseCurrency;
+  let quoteName = document.getElementById("quote-name").value.toLowerCase();
+  let quoteRate = document.getElementById("quote-rate").value;
+  brandObject.rates[quoteName] = +quoteRate;
+
+  document.getElementById("quote-name").value = "";
+  document.getElementById("quote-rate").value = "";
+  console.log({ brandObject });
+  console.log("List of inserted currency from Brand Currency Form:");
+  console.log(brandObject.rates);
+}
+
+// Currency Converter form --------------
+
+document
+  .getElementById("valuta-converter-form")
+  .addEventListener("submit", valutaConverter);
+
+function valutaConverter(event) {
+  event.preventDefault();
+
+  let convertedAmount;
+
+  const moneyAmount = +document.getElementById("money-amount").value;
+  const chosenQuoteValuta = document
+    .getElementById("valuta-exchange-quote-valuta")
+    .value.toLowerCase();
+
+  if (brandObject.rates[chosenQuoteValuta]) {
+    //Checks if the guote valuta is available
+    convertedAmount = moneyAmount * brandObject.rates[chosenQuoteValuta];
+  } else {
+    alert("Conversion rate not available, insert another valuta");
+  }
+  document.getElementById("converted-amount-value").innerHTML =
```

Comment:
This must be in the first statement of the condition

```
  if (brandObject.rates[chosenQuoteValuta]) {
    //Checks if the guote valuta is available
    convertedAmount = moneyAmount * brandObject.rates[chosenQuoteValuta];
  }
```

otherwise, it will throw error if the rate is not valid `Uncaught TypeError: Cannot read properties of undefined (reading 'toFixed')`

Reply:
Sorry, I couldn't understand what I should change. 
I have added .toLowerCase() at the end of the values in the first form and the second form.
And now I have not gotten that error again.
const chosenQuoteValuta = document
    .getElementById("valuta-exchange-quote-valuta")
    .value.toLowerCase();

Reply:
`document.getElementById("converted-amount-value").innerHTML = convertedAmount.toFixed(2);`

This part must be moved into 
` if (brandObject.rates[chosenQuoteValuta]) {`

like this :) 
```
 if (brandObject.rates[chosenQuoteValuta]) {
    //Checks if the guote valuta is available
    convertedAmount = moneyAmount * brandObject.rates[chosenQuoteValuta];
    document.getElementById("converted-amount-value").innerHTML = convertedAmount.toFixed(2);
  }
```

Reply:
@katayouny please do this and we can merge it @katayouny